src/Symfony/Component/Console/Tests/Input/StringInputTest.php

```diff
@@ -52,7 +52,8 @@ public function testLegacyInputOptionDefinitionInConstructor()
             array(new InputOption('foo', null, InputOption::VALUE_REQUIRED))
         );
 
-        $input = new StringInput('--foo=bar', $definition);
+        $input = new StringInput('--foo=bar');
+        $input->bind($definition);
```

Comment:
You can remove the whole test (it's the same as `testInputOptionWithGivenString` above now).